src/utils/Label.js

```diff
@@ -5,14 +5,36 @@ import tail from "lodash/tail";
 import min from "lodash/min";
 import max from "lodash/max";
 import reduce from "lodash/reduce";
-import moment from "moment";
 import numeral from "numeral";
+import { timeFormat } from "d3";
+
+export function getDefaultFormats(scaleType) {
+  const defaultTimeFormats = ["%Y", "'%y", "%b %Y", "%m/%Y"];
```

Comment:
I don't know d3's timeFormat but I'm assuming these are the same as Moment's `["YYYY", "'YY", "MMM YYYY", "M/YY"]` ?

Reply:
Yup, that's correct!